swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
```

Comment:
Great start on using the Item super class here! However, this doesn't fully take advantage of what super classes can do! You don't need to story category or condition information here, you can pass that up to the Item class. None of the child classes need those lines of code, which will make the code cleaner.